conf/test.config

```diff
@@ -15,15 +15,88 @@ params {
     config_profile_description = 'Minimal test dataset to check pipeline function'
 
     // Limit resources so that this can run on GitHub Actions
-    max_cpus   = 2
-    max_memory = '6.GB'
-    max_time   = '6.h'
+    max_memory = "2.GB"
+    max_cpus = 1
+    max_time   = '3.h'
 
     // Input data
-    // TODO nf-core: Specify the paths to your test data on nf-core/test-datasets
-    // TODO nf-core: Give any required params for the test so that command line flags are not needed
-    input  = 'https://raw.githubusercontent.com/nf-core/test-datasets/viralrecon/samplesheet/samplesheet_test_illumina_amplicon.csv'
+    input = "https://raw.githubusercontent.com/phac-nml/mikrokondo/dev/tests/data/samplesheets/samplesheet-small-assembly.csv"
+    outdir = "results"
 
-    // Genome references
-    genome = 'R64-1-1'
+    platform = "illumina"
+
+    mash {
```

Comment:
Thank so much for looking into this and including these parameters.

Have you tried only setting the parameters you need with dotted notation? As in:

```
mash.mash_sketch = "https://github.com/phac-nml/mikrokondo/raw/dev/tests/data/databases/campy-staph-ecoli.msh"
```

Could you maybe try this out, to see if we can avoid having duplicate copies of all parameters here (that may become out-of-sync)?

Alternatively, I wonder if the full test profile itself could remove the `params { }` block and set everything with dotted notation. As in:

```
params.mash.mash_sketch = "https://github.com/phac-nml/mikrokondo/raw/dev/tests/data/databases/campy-staph-ecoli.msh"
```

Reply:
Oh sure. I can try it and see if it works.

Reply:
Yup, you were right. It works. Updated in: 865fe4dfacdb6a998c581462e47205ff09b3cbd6

Reply:
Awesome. Thanks so much 😄